ap2-receiver.py

```diff
@@ -480,7 +480,7 @@ def do_SET_PARAMETER(self):
         elif content_type == HTTP_CT_IMAGE:
             if content_len > 0:
                 fname = None
-                with tempfile.NamedTemporaryFile(prefix="artwork", dir=".", delete=False) as f:
+                with tempfile.NamedTemporaryFile(prefix="artwork", suffix=".jpg", dir=".", delete=False) as f:
```

Comment:
Can we move this to a different PR please? (it's not related to listing network interfaces ;)
Airplay does not convert media types (to e.g. send only a jpg), and the receiver must be able to decode multiple file types. Most pics are probably sensible as a jpg, but there is media out there which is not jpg...

Reply:
Content type is jpg not image

I miss the different pr comment. 

Do you squash merge?

Reply:
Sorry, I forgot that the `HTTP_CT_IMAGE` var is set to jpg. 🤦  But airplay is designed to receive other types, if I recall correctly.